5EHIFClassProjectIngredientsUnitTests/InsertDataCSVUnitTest.cs

```diff
@@ -1,6 +1,72 @@
+using Allergens.Database;
+using Ingredients.Database;
+using Ingredients.Model;
+using Moq;
+using Neo4j.Driver;
+using System.Text;
+
 namespace _5EHIFClassProjectIngredientsUnitTests;
 
 public class InsertDataCSVUnitTest // todo Sophie
 {
-    
+
+    [Fact]
+    public void ReadCsvFile_ReturnsCorrectNumberOfItems()
+    {
+        // Arrange
+        var mockDriver = new Mock<IDriver>();
+        var insertDataCSV = new InsertDataCSV(mockDriver.Object);
+        var csvContent = new StringBuilder()
+            .AppendLine("header1,header2,header3,header4,header5,header6,header7,header8,header9,header10,header11,header12,header13,header14,header15,header16,header17,header18,header19,header20,header21,header22,header23,header24,header25,header26,header27,header28,header29")
+            .AppendLine("\"value1\",\"value2\",\"value3\",\"value4\",\"value5\",\"value6\",\"7\",\"value8\",\"value9\",\"value10\",\"value11\",\"value12\",\"13\",\"value14\",\"value15\",\"value16\",\"value17\",\"value18\",\"value19\",\"value20\",\"value21\",\"value22\",\"value23\",\"value24\",\"value25\",\"value26\",\"value27\",\"value28\",\"29\"").ToString();
+
```

Comment:
This might look ugly and it is but it also serves it purpose. IMO its actually a good way of solving the problem.